packages/datadog-plugin-mysql/src/index.js

```diff
@@ -55,4 +56,14 @@ class MySQLPlugin extends Plugin {
   }
 }
 
+function getServiceName (tracer, config) {
+  if (typeof config.service === 'function') {
+    return config.service()
```

Comment:
Should something be passed to this function? Otherwise, how can it decide what servie name to use?

Reply:
Good call. In our case we are using async hooks to know the context we're in. But it makes more sense to simply pass the relevant config into the service function...updated. 👍